.travis.yml

```diff
@@ -18,13 +18,17 @@ install:
   # Install dependencies
   - conda create -n test-environment python=$TRAVIS_PYTHON_VERSION
   - source activate test-environment
-  - conda install pytest numpy pip coverage toolz pandas cython scikit-learn cytoolz dill networkx pyzmq psutil
+  - conda install pytest numpy pip coverage toolz pandas cython scikit-learn cytoolz dill networkx pyzmq psutil ipython
   - pip install chest
   - pip install git+https://github.com/blosc/bcolz --upgrade
 
   # Install dask
   - python setup.py install
 
+  # start the ipcluster, it takes a while to start
+  - ipcluster start -n 4 &
+  - sleep 20
```

Comment:
I don't think we need to worry about the state of the Travis-CI box when we are done? I can't think of any problems it would cause. 

I `sleep 20` seems kind of hacky but last time I did this there was no return value from ipcluster when it was ready. So I had to parse stdout looking for "Engines appear to have started successfully" or something. 

I was considering building more tools around working with `ipcluster`, but realized I didn't really need them since this is all we need to do here.

Something more sophisticated could be useful when we are doing more extensive distributed testing.